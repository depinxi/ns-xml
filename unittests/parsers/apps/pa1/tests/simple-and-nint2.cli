-- "The simple positional argument... and a non-positive integer value" 0 