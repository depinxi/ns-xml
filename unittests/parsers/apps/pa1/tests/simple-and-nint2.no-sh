CLI: "--", "The simple positional argument... and a non-positive integer value", "0"
Value count: 2
Values: "The simple positional argument... and a non-positive integer value", "0"
Error count: 1
Subcommand: 
