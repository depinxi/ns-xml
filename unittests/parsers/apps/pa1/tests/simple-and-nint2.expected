CLI: "--", "The simple positional argument... and a non-positive integer value", "0"
Value count: 1
Values: "The simple positional argument... and a non-positive integer value"
Error count: 1
Subcommand: 
